src/client.ts

```diff
@@ -15,7 +15,10 @@
 
 import { IEnhancedMetric } from './types';
 
-const URL = process.env.SERVICE_URL as string;
+export const send = (events: IEnhancedMetric[]): void => {
+  const { SERVICE_URL } = process.env;
```

Comment:
I know that Webpack has trouble replacing environment variables when they're destructured, because it does a simple string replacement. Do you know if Rollup has the same limitation? I assume you've tested this and it works?

Reply:
I'll take it for a test ride in the browser now (I know I can also check the bundle)

Reply:
@connor-baer tested and it's working as expected